operations/rack_resiliency/rack_resiliency_cli.md

```diff
@@ -0,0 +1,131 @@
+# Rack Resiliency Service CLI Commands
+
+This page is a quick reference for common RRS commands in the [Cray CLI](../../glossary.md#cray-cli-cray).
+
+To find the API versions of any commands listed, add `-vvv` to the end of the CLI command, and the CLI will print the underlying call to the API in the output.
+For more information about the RRS API, see [Rack Resiliency Service](../../api/rrs.md).
+
+* [Managing zones](#managing-zones)
+* [Managing critical services](#managing-critical-services)
+* [Monitoring critical services status](#monitoring-critical-services-status)
+* [Updating critical services](#updating-critical-services)
+
+## Managing zones
+
+* (`ncn-mw#`) List all configured zones:
+
+    ```bash
+    cray rrs zones list
+    ```
+
+* (`ncn-mw#`) Get detailed information about a specific zone:
+
+    ```bash
+    cray rrs zones describe <zone-name>
+    ```
+
+  This will return detailed information about the zone including management master nodes, worker nodes, and storage nodes with their status.
+
+## Managing critical services
+
+* (`ncn-mw#`) List all critical services grouped by namespace:
+
+    ```bash
+    cray rrs criticalservices list
+    ```
+
+* (`ncn-mw#`) Get summarized information about a specific critical service:
+
+    ```bash
+    cray rrs criticalservices describe <critical-service-name>
+    ```
+
+  This will return information such as configured instances, currently running instances, name, namespace, and type.
+
+## Monitoring critical services status
+
+* (`ncn-mw#`) List the status of all critical services with distribution details:
+
+    ```bash
+    cray rrs criticalservices status list
+    ```
+
+  This will show service status (e.g., balanced, no pods found) and distribution of pods across zones and nodes.
+
+* (`ncn-mw#`) Get detailed status information about a specific critical service:
+
+    ```bash
+    cray rrs criticalservices status describe <critical-service-name>
+    ```
+
+  This will return detailed pod information including pod names, nodes, statuses, and zones.
+
+## Updating critical services
+
+* (`ncn-mw#`) Update the critical services configuration using a JSON file:
+
+    ```bash
+    cray rrs criticalservices update --from-file <file-path>
+    ```
+
+  The file should contain a JSON string with the critical services configuration in the following format:
+
+    ```json
+    {
+      "critical-services": {
+        "coredns": {
+          "namespace": "kube-system",
+          "type": "Deployment"
+        },
+        "kube-proxy": {
+          "namespace": "kube-system",
+          "type": "DaemonSet"
+        }
+      }
+    }
+    ```
+
+  This format allows you to specify multiple services together with their namespaces and types.
+
+## Examples
```

Comment:
It would probably be more usable to have each example paired up with its corresponding command, rather than have all of the commands documented abstractly, and then all of the examples.